### DISCUSSION TITLE
Can't find "entropy-cli.config" config file

### DISCUSSION THREAD

**Discussion by alinabokaya** [entropyxyz/community#68] [Support] [answered]
Hi! 
I have an account on my other PC and now I can't import it to another PC because can't recover the seed phrase from entropy-cli.config file. This file just doesn't exist on my PC. 

Even when I am creating the new account there is no "type: seed" field with generated seed, like on this example:
 
<img width="677" alt="341200794-8f3793e8-7397-42f3-a9dd-44b330f95f11" src="https://github.com/entropyxyz/community/assets/106039036/5d56fc43-8073-4740-a1d3-f82026eaee50">

**Comment by johnnymatthews** [comment]
> This file just doesn't exist on my PC.

On the PC that seemingly doesn't have the `entropy-cli.config` file, what happens when you open the CLI, go to **Manage Accounts** and select **List Accounts**?

**Comment by alinabokaya** [comment]
Hi! @johnnymatthews I see this: 

<img width="570" alt="image" src="https://github.com/entropyxyz/community/assets/106039036/e2d7f46d-1d1a-4ada-befe-1ff0e4dcab76">

**Comment by johnnymatthews** [comment]
I've realized that the screenshot you shared in your original post is showing what an older version of the CLI did. There was a change to the CLI recently where the `type: seed` line was removed; we just hadn't updated the docs yet to reflect this change. I've just made those changes in https://github.com/entropyxyz/entropy-docs/pull/140. 

What happens when you enter `cat ~/.entropy-cli.config` into your terminal? You _should_ get something like this:

```
cat ~/.entropy-cli.config
{"accounts":[{"name":"Yoel","address":"5GUARejgGqFxERGih3D8PdeNE3uqrmBVVUjffkVUHxRtuMXA","data":{"debug":true,"seed":"0x74d1b1639djiej2983jd8k4ie8w3203085e93i209485792ea84b3dee67bd5837","admin":{"address":"5GUARejgGqFxERGih3D8PdeNE3uqrmBVVUjffkVUHxRtuMXA","type":"registration","verifyingKeys":[],"userContext":"ADMIN_KEY","seed":"0x74d1b16481c91eaff54d946d7423203085ef8bd9f15a076f037a052e67bd5837","path":"","pair":{"address":"5GUARejgGqFxERGih3D8PdeNE3uqrmBVVUjffkVUHxRtuMXA","addressRaw":{"0":194,"1":213,"2":4,"3":38,"4":216,"5":175,"6":169,"7":242,"8":123,"9":136,"10":115,"11":210,"12":107,"13":46,"14":150,"

...
```

If you get something like that, then your `seed` variable is found towards the top of that chunk of text.

If, however, you get something like `cat: /home/io/.entropy-cli.config: No such file or directory` then there's something wrong with your CLI installation, and we've have to do a bit more digging.

**Reply by alinabokaya** [reply, marked as answer]
Thanks! 
Command `cat ~/.entropy-cli.config` helped me to find my seed.
